src/uu/stdbuf/src/libstdbuf/src/libstdbuf.rs

```diff
@@ -52,7 +54,11 @@ fn set_buffer(stream: *mut FILE, value: &str) {
         res = libc::setvbuf(stream, buffer, mode, size);
     }
     if res != 0 {
-        crash!(res, "error while calling setvbuf!");
+        eprintln!(
+            "could not set buffering of {} to mode {}",
+            unsafe { fileno(stream) },
```

Comment:
Can we do this with `RawFd` as well? A pure Rust safe solution makes a bit more sense I think.

Reply:
Nevermind, this is of course already a `libc::FILE` so `RawFd` doesn't work :)